main.py

```diff
@@ -7,6 +7,6 @@
 faulthandler.enable(open('crash_log.txt', 'w'))
 root = Tk()
 root.wm_title("The Eternal City")
-root.wm_iconbitmap(root, resource_filename('resources.images', 'eternal_logo.ico'))
+#root.wm_iconbitmap(root, resource_filename('resources.images', 'eternal_logo.ico'))
```

Comment:
Any idea why this crashes it on Ubuntu? Do I just need to convert it to another format? Sounds like there should be an issue for this.